docs/geedocs/answer/3481558.html

```diff
@@ -127,6 +127,9 @@
 <code><b>--editvolumes</b> [--assetroot <i>path</i>]</code></pre>
 <p><i>Optional</i>. Follow the prompts to add a volume to the selected asset root or, modify the <code>localpath</code> definition for an existing volume, or to add a volume definition.</p>
 <pre>
+<code><b>--listvolumes</b> [--assetroot <i>path</i>]</code></pre>
+<p><i>Optional</i>. List the available (configured) volumes for the selected asset root.</p>
```

Comment:
Perhaps, we should mention that only the volumes on the host the command is executed from will be shown.